src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,12 @@
-import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => <PostInfo key={post.id} post={post} />)}
```

Comment:
In my opinion, this looks a little bit better
```suggestion
    {posts.map(post => (
      <PostInfo key={post.id} post={post} />
    ))}
```